web/templates/tooltip_templates.hbs

```diff
@@ -131,23 +133,33 @@
 <template id="inbox-tooltip-template">
     <div class="views-tooltip-container" data-view-code="inbox">
         <div>{{t 'Inbox' }}</div>
+        <div class="tootlip-inner-content views-message-count italic"></div>
         <div class="tooltip-inner-content views-tooltip-home-view-note italic hide">{{t 'This is your home view.' }}</div>
     </div>
     {{tooltip_hotkey_hints "Shift" "I"}}
 </template>
 <template id="drafts-tooltip-template">
-    {{t 'Drafts' }}
+    <div class="views-tooltip-container" data-view-code="drafts">
+        <div>{{t 'Drafts' }}</div>
+        <div class="tootlip-inner-content views-message-count italic"></div>
+    </div>
     {{tooltip_hotkey_hints "D"}}
 </template>
+<template id="scheduled-tooltip-template">
+    <div class="views-tooltip-container" data-view-code="scheduled_message">
+        <div>{{t 'Scheduled messages'}}</div>
+        <div class="tootlip-inner-content views-message-count italic"></div>
+    </div>
+</template>
 <template id="show-all-direct-messages-template">
     {{t 'Direct message feed' }}
     {{tooltip_hotkey_hints "Shift" "P"}}
 </template>
 <template id="mentions-tooltip-template">
-    {{t 'Mentions' }}
-</template>
-<template id="starred-tooltip-template">
```

Comment:
The condensed starred messages tooltip still refers to the "starred-tooltip-template", which results in an empty tooltip when the template is removed. Let's update that to use the "starred-message-tooltip-template" tooltip template.

![Screenshot 2024-09-20 at 9 15 48 PM](https://github.com/user-attachments/assets/148ea908-c260-448e-b2d8-2901cfd79a95)

Reply:
> The condensed starred messages tooltip still refers to the "starred-tooltip-template"

I think this was missed in a recent change from `starred-tooltip-template` to `starred-message-tooltip-template`. In any case, I have updated the reference to the correct tooltip template